src/Chunk.ts

```diff
@@ -142,9 +142,9 @@ export default class Chunk {
 		dependencies: ChunkDependencies;
 		exports: ChunkExports;
 	} = undefined;
-	isEmpty: boolean;
+	private chunkName: string | void;
```

Comment:
Wasn't the `chunk.id` the same as the `chunkName` before? How are these two properties different?

Reply:
chunkName is just the "[name]" part of  the chunk id and is also used by `generateId()` for this purpose. The idea is that the name can be used as a stable way to identify a chunk while the actual id depends on the name pattern and possibly the hash. This name is now attached to the output.